core/src/main/codegen/templates/Parser.jj

```diff
@@ -8750,9 +8759,10 @@ MORE :
     < IDENTIFIER: <LETTER> (<LETTER>|<DIGIT>)* >
 }
 
-<DEFAULT, DQID, BTID, BQID> TOKEN :
+<DEFAULT, DQID, BTID, BQID, BQHID> TOKEN :
```

Comment:
Can you explain this change?

Reply:
Leftover from debuging and didn't revert, good catch.